src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+const ROOT_URL='https://yts.am/api/v2/list_movies.json';
+
+export const SEARCH_MOVIES='SEARCH_MOVIES';
+export const POPULAR_DOWNLOADS='POPULAR_DOWNLOADS';
+export const LATEST_TORRENTS='LATEST_TORRENTS';
+
+export function SearchMovies(movieTerm) {
+  const request= axios.get(`${ROOT_URL}?limit=5&query_term=${movieTerm}`);
```

Comment:
Limit is hardcoded here, you should have it as an attribute of the function, and you can set the default like so:
searchMovies({movieTerm, limit = 5})
Also the function name starts with capital letter, change to camelCase style.